src/test/java/uk/gov/service/notify/PdfUtilsTest.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.service.notify;
+
+import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.io.FileUtils;
+import org.junit.Test;
+
+import java.io.File;
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.Assert.*;
+
+public class PdfUtilsTest
+{
+    @Test
+    public void testIsFilePDFValidPdf() throws Exception
+    {
+        ClassLoader classLoader = getClass().getClassLoader();
+        File file = new File(classLoader.getResource("one_page_pdf.pdf").getFile());
+        assertTrue(PdfUtils.isFilePDF(file));
+    }
+
+    @Test
+    public void testIsFilePDFNotValidPdf() throws Exception
+    {
+        ClassLoader classLoader = getClass().getClassLoader();
+        File file = new File(classLoader.getResource("not_a_pdf.txt").getFile());
+        assertFalse(PdfUtils.isFilePDF(file));
+    }
+
+    @Test(expected = NotificationClientException.class)
+    public void testIsFilePDFThrowException() throws Exception
+    {
+        File file = new File("this is a path");
+        assertFalse(PdfUtils.isFilePDF(file));
```

Comment:
Do you need assertFalse if expecting an exception?

Reply:
done